docs/configuration.rst

```diff
@@ -347,8 +347,13 @@ AUTO_*
 ~~~~~~~~~~~~
 
 These configurations indicates that thumbor will try to automatically convert
-the image format to a lighter image format, following a better compression order
-`WEBP -> AVIF -> JPG -> HEIF`.
+the image format to a lighter image format, following a better compression order.
+The order in which formats will be tried (if enabled and possible):
+
+1. WEBP
+2. AVIF
+3. JPG
+4. HEIF
```

Comment:
I'd say this may be enough:

```suggestion
the image format to a lighter image format, according to this compression order:
`WEBP, AVIF, JPG, HEIF` — from highest (`WEBP`) to lowest (`HEIF`) priority.
```

The following sections touch on "enabled" and "possible" for each of those formats.

Reply:
@scorphus thanks! 😄